### DISCUSSION TITLE
Acer Nitro 5 AN515-57 98VC

### DISCUSSION THREAD

**Discussion by myxzlpltk** [itsmeshibintmz/Acer-Nitro-5-AN515-54-Monterey#11] [Q&A] [answered]
Hi, just want to ask. is this going to work with Acer Nitro 5 AN515-57 98VC? Thank you

**Comment by itsmeshibintmz** [comment, marked as answer]
Sorry, it won't. This EFI supports AN515-54 series only.

**Reply by myxzlpltk** [reply]
Had to find out the hard way :(

![IMG-20220628-WA0009.jpg](https://user-images.githubusercontent.com/59659668/176206558-d737e8aa-ba8e-404a-b60e-1a991e6056ff.jpg)
